source/index.md

```diff
@@ -2008,6 +2008,92 @@ The user can list all SoftLayer Cloud Server plans accessing the route bellow.
 
 Route: *`GET "/api/v1/shop-softlayer-cloud-server-plans"`*
 
+# User Shop Price
+
+When logged in, the user can access information about the shop. This access allows him to create shop prices.
```

Comment:
Create shop prices? It's a POST route, but user do not create `shop prices`, this is used when users want to verify a price for a custom plan.